src/Stratis.Features.FederatedPeg/Collateral/SmartContractCollateralPoARuleRegistration.cs

```diff
@@ -28,11 +20,11 @@ public class SmartContractCollateralPoARuleRegistration : SmartContractPoARuleRe
 
         private readonly IDateTimeProvider dateTime;
 
-        public SmartContractCollateralPoARuleRegistration()
-        : base()
+        public SmartContractCollateralPoARuleRegistration() : base()
         {
         }
 
+        [NoTrace]
```

Comment:
Shouldn't this only be getting done once anyway?

Reply:
I supposed so but yeah its not really needed anyway? I can remove it?

Reply:
Probably not worth the additional test run